galter_subjects_utils/contrib/lcsh/converter.py

```diff
@@ -106,6 +109,23 @@ def is_deprecated(topic):
     return bool(find_deprecation_node(topic, node_main))
 
 
+def happened_since(topic, since=None):
+    """Filter for topic having happened after since.
+
+    :param topic: skos topic
+    :type since: dict
```

Comment:
change to ":type topic: dict"

Reply:
I just rebased it, so those should go away with the fix from the previous PR :wink: